### ISSUE TITLE
Remove columns from Detailed Usage Report

### ISSUE THREAD

**Issue by ferristocrat** [storj/storj#6498]
### Summary
The "Detailed Usage Report" contains columns that we shouldn't pass along to customers as they are not relevant.

The download link can be found at https://us1.storj.io/account/billing/overview:

![Image](https://github.com/storj/storj/assets/7063002/41140c6b-30d5-4c05-935d-a951b8ad761f)

### Acceptance Criteria
- The Detailed Usage Report no longer contains the following columns:
  - `MetadataSize GB-hour`
  - `RepairEgress GB`
  - `AuditEgress GB`

**Comment by ferristocrat** [comment]
Question on `TotalSegments GB-hour` and `ObjectCount GB-hour`: why aren't they integers/whole numbers? The GB-hour part doesn't really make sense to me... or is this some average over the time period? I guess it could make sense for segments if that's the number we're using for billing, but object count seems to lose any value as a fraction.

**Comment by VitaliiShpital** [comment]
@ferristocrat this is how calculation is done back from 2019. So we sum up `value*hours` of a set of bucket storage tallies where `hours` is hour diff between 2 closest tallies. This looks weird for me as well
```
        // fill metadata, objects and stored data
	// hours calculated from previous tallies,
	// so we skip the most recent one

	for i := len(bucketStorageTallies) - 1; i > 0; i-- {
		current := bucketStorageTallies[i]

		hours := bucketStorageTallies[i-1].IntervalStart.Sub(current.IntervalStart).Hours()

		if current.TotalBytes > 0 {
			bucketRollup.TotalStoredData += memory.Size(current.TotalBytes).GB() * hours
		} else {
			bucketRollup.TotalStoredData += memory.Size(current.Remote+current.Inline).GB() * hours
		}
		bucketRollup.MetadataSize += memory.Size(current.MetadataSize).GB() * hours
		if current.TotalSegmentsCount > 0 {
			bucketRollup.TotalSegments += float64(current.TotalSegmentsCount) * hours
		} else {
			bucketRollup.TotalSegments += float64(current.RemoteSegmentsCount+current.InlineSegmentsCount) * hours
		}
		bucketRollup.ObjectCount += float64(current.ObjectCount) * hours
	}
```

**Comment by mobyvb** [comment]
saying GB-hour is similar to Watt-hours (commonly seen as kWh) as a unit. At any moment in time, the total object count can change. So if we take a snapshot one hour, the user may have 20GB stored. Then the next hour, the user may have 10GB stored. An equation to describe this situation is `20GB * 1 hour + 10GB * 1 hour = 30 GB-hour`. This unit allows us to describe the total amount of storage the user has consumed, even though the "rate" of usage (object count) can change throughout the timeline
Basically GB-hour is the shaded area below the line on a graph of usage vs. time (integral)
